backend/requirements.in

```diff
@@ -16,3 +16,5 @@ gunicorn
 pillow
 psycopg2-binary
 python-dotenv
+pyclamd

```

Comment:
Ah, ok. I see here that the clamav modules are included in requirements text file.